pkgs/development/python-modules/gtimelog/default.nix

```diff
@@ -1,27 +1,43 @@
-{ lib, fetchFromGitHub, makeWrapper
-, glibcLocales, gobject-introspection, gtk3, libsoup, libsecret
-, buildPythonPackage, python
-, pygobject3, freezegun, mock
+{ lib
+, fetchFromGitHub
+, makeWrapper
+, python
+, buildPythonPackage
+, atk
+, gdk-pixbuf
+, glibcLocales
+, glib-networking
+, gobject-introspection
+, gtk3
+, harfbuzz
+, libsecret
+, libsoup
+, pango
 }:
-
 buildPythonPackage rec {
   pname = "gtimelog";
-  version = "unstable-2020-05-16";
+  version = "git-2022-10-28";
```

Comment:
By [convention](https://nixos.org/manual/nixpkgs/stable/#sec-package-naming), it's `unstable-` for non tagged releases. But it's fine for me.